lib/modules/versioning/ubuntu/index.ts

```diff
@@ -54,8 +66,31 @@ function isStable(version: string): boolean {
 
 // digestion of version
 
+function getDatedContainerImageCodename(version: string): null | string {
+  const groups = /^(\w+)-(\d{8})$/.exec(version);
+  if (groups === null) {
+    return null;
+  }
+  return groups[1];
+}
+
+function getDatedContainerImageVersion(version: string): null | number {
+  const groups = /^(\w+)-(\d{8})$/.exec(version);
+  if (groups === null) {
+    return null;
+  }
+
+  return parseInt(groups[2]);
```

Comment:
```suggestion
  return parseInt(groups[2], 10);
```

enforce  base 10 parsing, otherwise it can have unwanted side effect if it's starting with a zero

Reply:
done.